src/handlers/backfill.ts

```diff
@@ -165,15 +165,7 @@ async function updateMissingSendPacketFees(ctx: Context) {
     if (sendPacket) {
       sendPacket.totalRecvFeesDeposited = sendPacket.totalRecvFeesDeposited + BigInt(sendPacketFee.recvGasLimit * sendPacketFee.recvGasPrice);
       sendPacket.totalAckFeesDeposited = sendPacket.totalAckFeesDeposited + BigInt(sendPacketFee.ackGasLimit * sendPacketFee.ackGasPrice);
-      if (!sendPacket.feesDeposited) {
-        sendPacket.feesDeposited = [sendPacketFee];
-      }
-      // Store up to 20 fee transactions per packet
-      else if (sendPacket.feesDeposited.length >= 20) {
```

Comment:
`feesDeposited` are not stored as an array in the table, it's 1:n connection represented as an array in the code.

Reply:
So you don't want to limit the size of this at all?

Reply:
There is no need